docs/configs/bebop.py

```diff
@@ -0,0 +1,43 @@
+from parsl.addresses import address_by_hostname
+from parsl.launchers import SrunLauncher, SingleNodeLauncher
+from parsl.providers import SlurmProvider
+
+from funcx_endpoint.endpoint.utils.config import Config
+from funcx_endpoint.executors import HighThroughputExecutor
+from funcx_endpoint.strategies import SimpleStrategy
+# fmt: off
+
+# PLEASE UPDATE user_opts BEFORE USE
+user_opts = {
+    'bebop': {
+        'worker_init': '',
+        'scheduler_options': '#SBATCH --ntasks-per-node=10',
+    }
+}
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            max_workers_per_node=10,
+            strategy=SimpleStrategy(max_idletime=300),
```

Comment:
Unless there's a reason, we should skip defaults from the example configs.

Reply:
Also, what's the purpose of the SimpleStrategy here? I didn't think that's Bebop specific, and we perhaps remove it to simply to the bare minimum needed to understand Bebop.

Reply:
The simple strategy keeps the node warm for 300s after a task completes. I'm fine with removing that too. I see it's also in the polaris config. We should likely standardize them and remove defaults elsewhere.